index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Sharon Odhiambo</title>
+</head>
+
+<body class="header-background">
+  <header class="nav">
+    <a href="#" id="logo">My logo</a>
+    <nav>
+      <ul class="nav-menu" id="nav-menu">
+        <li class="nav-link"><a href="#works">Portfolio</a></li>
+        <li class="nav-link"><a href="#about-myself">About</a></li>
+        <li class="nav-link"><a href="#form-section">Contact</a></li>
+      </ul>
+      <div class="mobile-navbar" id="mobile-navbar">
+        <div id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul class="mobile-nav-menu" id="mobile-nav-menu">
+          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
+          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="headline" id="headline"> 
+    <img src="./images/header-bg.svg" alt="Header bg">
+    <div class="headline-content">
+      <h1>I'm Sharon, <br> Glad to see you!</h1>
+      <p>I’m a software developer! I can help you build a product , feature or website <br> 
+        Look through some of my work and experience. <br> If you like what you see and have a project you need coded, don’t hestiate to contact me!
+      </p>
+      <h2 class="lets-connect">LET’S CONNECT</h2>
+      <ul class="social-media-links">
+        <li> <a href="https://twitter.com/SharonVictor16_" target="_blank" rel="noopener" class="social-media-link">
+            </a>
+        </li>
+        <li> <a href="https://www.linkedin.com/in/sharon-odhiambo-4333a0163/" target="_blank" rel="noopener">
+            </a> 
+        </li>
+        <li> <a href="https://github.com/sharon-odhiambo" target="_blank" rel="noopener" class="social-media-link">
+            </a>
+        </li>
```

Comment:
- [ ] Also here, please update your project to match the Figma template,
your project: 
![Screenshot from 2022-05-27 16-20-38](https://user-images.githubusercontent.com/87965534/170707575-6c253ccf-4305-4d21-a607-d3a6584d6508.png)
Figma template: 
![Screenshot from 2022-05-27 16-19-49](https://user-images.githubusercontent.com/87965534/170707593-4da5cc11-730d-4840-aa8e-7e1b2b4f3447.png)